src/main/java/org/tikv/common/TiSession.java

```diff
@@ -166,6 +167,13 @@ private synchronized void warmUp() {
     long warmUpStartTime = System.nanoTime();
     BackOffer backOffer = ConcreteBackOffer.newRawKVBackOff();
     try {
+      // let JVM ClassLoader load gRPC error related classes
+      // this operation may cost 100ms
+      Errorpb.Error.newBuilder()
+          .setNotLeader(Errorpb.NotLeader.newBuilder().build())
```

Comment:
do we need to warm up other errors?

Reply:
All the classes (include other errors) in `org.tikv.kvproto.Errorpb` will be loaded during warmup.